code/modules/research/xenobiology/crossbreeding/_status_effects.dm

```diff
@@ -973,3 +973,87 @@ datum/status_effect/stabilized/blue/on_remove()
 				qdel(src)
 				qdel(linked_extract)
 	return ..()
+
+///////////////////////////////////////////////////////
+//////////////////DESTABILIZED EXTRACTS//////////////////
+///////////////////////////////////////////////////////
+
+/obj/screen/alert/status_effect/caged
+	name = "Caged"
+	desc = "You are trapped in a cage. Resist to escape."
+	icon_state = "slime_metal_cage"
+
+/datum/status_effect/cageTrapped
+	id = "slime_metal_cage"
+	status_type = STATUS_EFFECT_UNIQUE
+	duration = -1 //Will remove self when cage breaks.
+	alert_type = /obj/screen/alert/status_effect/caged
+	var/obj/structure/metalCage/cage
+
+/datum/status_effect/cageTrapped/on_apply()
+	RegisterSignal(owner, COMSIG_LIVING_RESIST, .proc/breakCage)
+	cage = new /obj/structure/metalCage(get_turf(owner))
+	owner.forceMove(cage)
+	return ..()
+
+/datum/status_effect/cageTrapped/tick()
+	if(!cage || owner.loc != cage)
+		owner.remove_status_effect(src)
+
+/datum/status_effect/cageTrapped/proc/breakCage()
+	if(do_after(owner, 450, FALSE, cage))
+		owner.remove_status_effect(src)
+
+/datum/status_effect/cageTrapped/on_remove()
+	if(cage)
+		qdel(cage)
+	UnregisterSignal(owner, COMSIG_LIVING_RESIST)
+
+/datum/status_effect/freon/destabilized
+	can_melt = FALSE
+
+/obj/screen/alert/status_effect/heavyBleeding
+	name = "Heavy Bleeding"
+	desc = "You are bleeding profusely. Only time can cure this."
+	icon_state = "slime_heavy_bleeding"
+
+/datum/status_effect/heavyBleeding
+	id = "heavy_bleeding"
+	duration = 150
+	alert_type = /obj/screen/alert/status_effect/heavyBleeding
+
+/datum/status_effect/heavyBleeding/tick()
+	if(iscarbon(owner))
+		var/mob/living/carbon/C = owner
+		if(ishuman(C))
+			var/mob/living/carbon/human/H = C
+			if(H.bleed_rate < 5)
+				H.bleed_rate = 5
+			if(H.bleedsuppress)
+				H.resume_bleeding()
```

Comment:
This might just be certain death. Not sure if that's balanced.

Reply:
Ive done some testing and while 15 seconds of bleeding is going to fuck you up, it still survivable. Comparative to other unstoppable damage options of even lower costs, this is fairly tame.

Reply:
Well from full health yea it is survivable, but if you are in a slime master vs John mcgreytide fight to the death,  slime master will probably bring several of these

Reply:
Sure, but again, if you're looking to damage someone in ways they have little-to-no options of preventing, there are a number of cheaper/more-easily-accessible and equally as effective options already in the game.